src/component/register/index.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react'
+import Button from '@material-ui/core/Button';
+import FormControl from '@material-ui/core/FormControl';
+import './register.css';
+import firebase from '../services/firebase.js'
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+
+import Handler from '../services/userOperation';
+
+let database = firebase.database().ref();
+let userData = database.child('users');
+
+export default class Register extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      loginState: '',
+
+    }
+  }
+
+  handleClick = () => {
+    this.setState({ open: true });
+  };
+
+  handleClose = (event, reason) => {
+    if (reason === 'clickaway') {
```

Comment:
@MayuriPapat25 : What does 'clickaway' mean ?  doesn't make any sense .

Reply:
sorry, it was from material-ui there is no use so I removed it.